src/NextGen/dataRepository.ts

```diff
@@ -0,0 +1,14 @@
+// QUESTION: Should we adopt BaseMessage like Native?
+import { BaseEvent } from '../sdkRuntimeModels';
+
+export class mParticleDataRepository {
+    // For demonstration purposes. This will likely go away as we spec this out
```

Comment:
Does `This will likely go away` refer just to the next line, or the general structure of the DataRepository?

Reply:
Just the next line. The Kotlin Spec doesn't have an internal state, but I added one just for testing.